packages/shorebird_cli/lib/src/commands/patch/patch_command.dart

```diff
@@ -179,20 +179,33 @@ NOTE: this is ${styleBold.wrap('not')} recommended. Asset changes cannot be incl
     final app = await codePushClientWrapper.getApp(appId: appId);
 
     File? patchArtifact;
-    final String releaseVersion;
+    final Release release;
     if (results.wasParsed('release-version')) {
-      releaseVersion = results['release-version'] as String;
+      final releaseVersion = results['release-version'] as String;
+      release = await codePushClientWrapper.getRelease(
+        appId: appId,
+        releaseVersion: releaseVersion,
+      );
```

Comment:
What is the behavior when this fails?

Reply:
`codePushClientWrapper.getRelease` logs an error and exits when it fails.